src/modules/processing/servicesPatternTesters.js

```diff
@@ -1,4 +1,6 @@
 export const testers = {
+    "videoclip": (patternMatch) => patternMatch.id?.match(/^[0-9]/),
```

Comment:
`^` in regex checks if it starts with, it will match `123_videotitle` , `123example`, `123 hi`, `123`
and with `[0-9]` it checks if it just starts with a number so it will match `1`, `123` and `123_videotitle`